src/timepicker/TimePickerColumnState.ts

```diff
@@ -8,7 +8,7 @@ export type ColumnType = "hour" | "minute" | "meridian";
 
 export interface TimePickerColumnInitialState extends ValueBase<Date> {
   visible?: boolean;
-  type?: ColumnType;
+  columnType?: ColumnType;
```

Comment:
Changed the `type` to `columnType` because jestdom is throwing erros with because native elements also have `type` prop.